desktop/main.js

```diff
@@ -82,13 +82,30 @@ const quitAndInstallWithUpdate = () => {
     autoUpdater.quitAndInstall();
 };
 
+/**
+ * Trigger event to show keyboard shortcuts
+ * @param {BrowserWindow} browserWindow
+ */
+const showKeyboardShortcutsModal = (browserWindow) => {
+    if (!browserWindow.isVisible()) {
+        return;
+    }
+    browserWindow.webContents.send('show-keyboard-shortcuts-modal');
+};
+
 // Defines the system-level menu item for manually triggering an update after
 const updateAppMenuItem = new MenuItem({
     label: 'Update New Expensify',
     enabled: false,
     click: quitAndInstallWithUpdate,
 });
 
+// Defines the system-level menu item for opening keyboard shortcuts modal
+const keyboardShortcutsMenu = new MenuItem({
+    label: 'View Keyboard Shortcuts',
```

Comment:
Is this a visible label? If so, this is not correct (and really the same can be said about the one right above this one). 
All texts need to be localized to the current user's language.

Reply:
it would be a bit of a challenge to add the localization to the Main process (native layer desktop) as Onyx data is stored on the renderer process UI layer. 
We can communicate between them. 

Crazy thought! should we parse the user locale from the OS timezone settings for the desktop app too?

Reply:
> We can communicate between them.

Let's do that then? Is it super hard or what?

> Crazy thought! should we parse the user locale from the OS timezone settings for the desktop app too?

I thought we already did, like where does JS's `Intl` get the data from? I assume that in the end, they come from the OS, no? Especially for the desktop app?

Reply:
> Let's do that then? Is it super hard or what?
Nope, But it seems reversed, For example, first the main app will be initialized then the UI layer so options will render in English and then switch from English to Selected language. But this time would be very short so it seems fine.

> I thought we already did, like where does JS's Intl get the data from? I assume that in the end, they come from the OS, no? Especially for the desktop app?

As far I can tell we don't parse that. we depend on the timezone from the user's profile which I think defaults to geolocation.

Reply:
I checked the other menu items like "Update New Expensify", "History", etc. they weren't localized. 

> We can communicate between them.

Can give it a try, will have to check that when the User changes the language preference, the system menu should update?

Reply:
Yes, it should, but the more I look at this the more I think we should do that in a separate issue.